### ISSUE TITLE
Biotin high in urine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11641]
**Preferred term label:**
Biotin high in urine

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
TPK1-Related Thiamine Metabolism Dysfunction Syndrome; see https://www.ncbi.nlm.nih.gov/books/NBK623293/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001542
- label: Elevated urinary biotin level
- parents: [Abnormal urinary organic compound level (HP:6000531)](https://hpo.jax.org/app/browse/term/HP:6000531)
- pmid: PMID:42335246
- def: The amount of biotin in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: This feature may be seen in TPK1-related thiamine metabolism dysfunction syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11641
- synonyms: Biotin high in urine(NONE)
